src/integrationTest/java/integration/tests/SystemEngineTest.java

```diff
@@ -177,27 +166,6 @@ void useSuccess() throws SQLException {
 		}
 	}
 
-	@Test
-	@Tag("v2")
-	@Tag("staging")
-	void useFailure() throws SQLException {
-		ConnectionInfo current = integration.ConnectionInfo.getInstance();
-		try (Connection connection = createConnection(getSystemEngineName())) {
-			try {
-				Assert.assertThrows(SQLException.class, () -> connection.createStatement().executeUpdate(format("USE %s", current.getDatabase()))); // unsupported and DB does not exist
-				Assert.assertThrows(SQLException.class, () -> connection.createStatement().executeUpdate(format("USE %s", USE_DATABASE_NAME))); // DB does not exist
-				connection.createStatement().executeUpdate(format("CREATE DATABASE IF NOT EXISTS %s", USE_DATABASE_NAME)); // create DB
-				Assert.assertThrows(SQLException.class, () -> connection.createStatement().executeUpdate(format("USE %s", USE_DATABASE_NAME))); // DB exists but use statement is unsupported anyway
-			} finally {
-				try (Statement statement = connection.createStatement()) {
-					statement.executeUpdate(format("DROP DATABASE %s", USE_DATABASE_NAME));
-				} catch (SQLException e) { // catch just in case to do our best to clean everything even if test has failed
-					log.warn("Cannot drop database ",  USE_DATABASE_NAME, e);
-				}
-			}
-		}
-	}
```

Comment:
Isn't this removal and `useSuccess` supposed to be other way around? Since we're removing dev, dev tests now can be deleted and staging kept by default.

Reply:
This should be and it is removed in other PR because staging now supports use statmenet

Reply:
I feel like this is just a matter of resolving merge conflicts correctly, but ok. Makes sense.